picoCTF-shell/setup.py

```diff
@@ -74,8 +74,9 @@
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
     install_requires=[
-        'spur', 'openssh-wrapper', "werkzeug", "jinja2", "pytest", "psutil",
-        "flask", "coloredlogs", "voluptuous"
+        'spur==0.3.20', 'openssh-wrapper==0.4', "werkzeug==0.14.1",
```

Comment:
Does the order of the items in this list matter? If not, let's (i) break the list down into one item per line and (ii) sort the list. This is better for version control.

Note that (ii) will lead to fixing the non-uniformity in the quotes within this list.